app/scripts/modules/titus/pipeline/stages/runJob/titusRunJobStage.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.titus.runJobStage', [
+  require('./runJobExecutionDetails.controller.js')
+])
+  .config(function (pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'runJob',
+      useBaseProvider: true,
+      cloudProvider: 'aws',
```

Comment:
the cloud provider here is aws so as to not require titus to be needed as a cloud provider. it's a bit of a hack.